Packs/IntSight/pack_metadata.json

```diff
@@ -1,8 +1,8 @@
 {
-    "name": "IntSights",
-    "description": "Use IntSights to manage and mitigate threats.",
+    "name": "Rapid7 - Threat Command (IntSights)",
+    "description": "Use Threat Command to manage alerts, CVEs, IOCs, and assets by accounts and MSSP accounts.",
     "support": "xsoar",
-    "currentVersion": "2.0.3",
+    "currentVersion": "3.0.0",
```

Comment:
we need a RN file `Packs/IntSight/ReleaseNotes/3_0_0.md` - use `demisto-sdk update-release-notes -g` to generate it.
FYI, not sure how it will behave - I assume you updated the version here manually, the `update-release-notes` cmd should take care of this too. If it doesn't work as expected, downgrade back to `2.0.3` and then run it again.